src/odemis/gui/cont/correlation.py

```diff
@@ -0,0 +1,280 @@
+
+        ### CONTROLS ##############################
+        # SHIFT + LEFT CLICK -> MOVE_TO_POSITION
+        # LEFT, RIGHT -> TRANSLATION X
+        # UP, DOWN -> TRANSLATION Y
+        # SHIFT + LEFT, RIGHT -> ROTATION
+        # SHIFT + UP, DOWN -> SCALE 
+        ###########################################
+
+        if _shift_mod:
+            dx, dy = 0, 0
+        else:
+            dr, dpx = 0, 0
+
+        logging.debug(f"key: {_key}, shift: {_shift_mod}")
+        logging.debug(f"dx: {dx}, dy: {dy}, dr: {dr}, dpx: {dpx}")
+
+        if _key == wx.WXK_LEFT:
+            self._move_stream(-dx, 0, -dr, 0)
+        elif _key == wx.WXK_RIGHT:
+            self._move_stream(dx, 0, dr, 0)
+        elif _key == wx.WXK_UP:
+            self._move_stream(0, dy, 0, dpx)
+        elif _key == wx.WXK_DOWN:
+            self._move_stream(0, -dy, 0, -dpx)
+    
+    def _move_stream(self, dx, dy, dr=0, dpx=0):
+        """move the selected stream by dx, dy, dr, dpx"""
+        if self.selectedStream.value is None:
+            return
+        
+        s = self.selectedStream.value
+    
+        logging.debug(f"move stream {s.name.value}: {dx}, {dy}, {dr}, {dpx}")
+
+        # translation
+        p = s.raw[0].metadata[model.MD_POS_COR]
+        if len(p) == 2:
+            x, y = p
+            s.raw[0].metadata[model.MD_POS_COR] = (x - dx, y - dy) # correlation direction is reversed
+        else:
+            x, y, z = p
+            s.raw[0].metadata[model.MD_POS_COR] = (x - dx, y - dy, z)
+
+        # rotation
+        rotation = s.raw[0].metadata.get(model.MD_ROTATION_COR, 0)
+        s.raw[0].metadata[model.MD_ROTATION_COR] = (rotation + dr)
+        
+        # scale (pixel size)
+        scalecor = s.raw[0].metadata.get(model.MD_PIXEL_SIZE_COR, (1, 1))
+        s.raw[0].metadata[model.MD_PIXEL_SIZE_COR] = (scalecor[0] + dpx, scalecor[1] + dpx)
+
+        # TODO: split x, y scale, add shear?
+
+        # update the image in the views
+        views = self._tab_data_model.views.value
+        update_image_in_views(s, views)
+        
+        # fit to content (TODO: these are not generic, might be different for other tabs)
+        try:
+            self._panel.vp_secom_tl.canvas.fit_view_to_content()
+            self._panel.vp_secom_tr.canvas.fit_view_to_content()
+        except AttributeError as e:
+            logging.debug(f"Requested views {e} are not availble to update...")
+    
+    def _move_stream_to_pos(self, pos: tuple):
+        """move the selected stream to the position pos"""
+        # the difference between the clicked position, and the position in metadata
+        # is the offset (be careful because correlation is sign flipped)
+        s = self.selectedStream.value
+        
+        # the cur_pos is the realspace position of the image
+        p = s.raw[0].metadata[model.MD_POS]
+        if len(p) == 2:
+            x, y = p
+        else:
+            x, y, z = p
```

Comment:
If you're never going to use `z`, you can replace this code by just:
```suggestion
        x, y = p[:2]
```